### PR TITLE
Enable fees

### PR DESCRIPTION
**Pull request by aqk**

### CODE REVIEW

File: chia/pools/pool_wallet.py [hunk 1]
```diff
@@ -462,14 +463,24 @@ async def create_new_pool_wallet_transaction(
...
         return standard_wallet_record, p2_singleton_puzzle_hash, launcher_coin_id
 
-    async def sign(self, coin_spend: CoinSpend) -> SpendBundle:
+    async def sign(self, coin_spend: CoinSpend, fee: uint64) -> SpendBundle:
         async def pk_to_sk(pk: G1Element) -> PrivateKey:
             owner_sk: Optional[PrivateKey] = await find_owner_sk([self.wallet_state_manager.private_key], pk)
-            assert owner_sk is not None
+            # assert owner_sk is not None
+            if owner_sk is None:
+                return self.standard_wallet.secret_key_store.secret_key_for_public_key(pk)
```
Comment (aqk) [inline chia/pools/pool_wallet.py:471]: This line is a hack to get the standard wallet key for signing.

Probably better to do this with `SpendBundle.aggregate`, as below

**Comment by aqk** [comment]
The intent of this patch is to test applying a fee to the operations in `PoolWallet`:

`create`, `join_pool`, `self_pool`, `claim_pool_rewards`

It does this in two ways:

* Assert that the fee is included in the generated SpendBundle
* Test that the wallet balance has the fee removed after the Transaction goes through

Some of the tests to NOT test that the wallet balance has the fee removed - see the absorb test.
The act of farming in that test puts the fee back into the wallet. I see two ways around that:

* Use a second full node to process the spend
* Spend away to a bad address when applying the spend - can't use this is absorb

File: chia/pools/pool_wallet.py (hunk 1 above)
Comment (mariano54) [reply chia/pools/pool_wallet.py:471]: Why can't we call `generate_signed_transaction`?

Comment (aqk) [reply chia/pools/pool_wallet.py:471]: Good idea.

**Pull request closed without merging**